CONTRIBUTING.md

```diff
@@ -1,163 +1,137 @@
 # Contributing to the Databricks Labs Data Generator
-We happily welcome contributions to *dbldatagen*. 
-
-We use GitHub Issues to track community reported issues and GitHub Pull Requests for accepting changes.
-
-## License
-
-When you contribute code, you affirm that the contribution is your original work and that you 
-license the work to the project under the project's Databricks license. Whether or not you 
-state this explicitly, by submitting any copyrighted material via pull request, email, or 
-other means you agree to license the material under the project's Databricks license and 
-warrant that you have the legal authority to do so.
-
-# Development Setup
-
-## Python Compatibility
-
-The code supports Python 3.10+ and has been tested with Python 3.10 and later.
-
-## Quick Start
-
+We happily welcome contributions to *dbldatagen*. GitHub Issues and Pull Requests are used to track issues and review changes.
```

Comment:
This is public, do we want to encourage external contributions before we confirm the process? I'm ok with it but perhaps we need to be more clear in the short term on what external contributors can expect if they submit a PR?